azext_iot/digitaltwins/providers/model.py

```diff
@@ -70,12 +68,17 @@ def __init__(self, cmd, name, rg=None):
         )
         self.model_sdk = self.get_sdk().digital_twin_models
 
-    def add(self, models=None, from_directory=None, failure_policy=ADTModelCreateFailurePolicy.ROLLBACK.value):
+    def add(self,
+            models=None,
+            from_directory=None,
+            failure_policy=ADTModelCreateFailurePolicy.ROLLBACK.value,
+            max_models_per_batch: int = None):
```

Comment:
I don't think currently a default of `None` works here for `max_models_per_batch`. If no value is provided, then `None` is used for the comparison and further logic on line 121/122. This should either have no default, or it can be typed as `Optional[int]` where if `None` is the value, the logic can add a default.

Reply:
got it, since I used MAX_MODELS_PER_BATCH as default for add_models, so the default is never used here, will remove